internal/controller/nnf_workflow_controller_test.go

```diff
@@ -623,6 +835,8 @@ var _ = Describe("NNF Workflow Unit Tests", func() {
 				deletePersistentStorageInstance(persistentStorageName)
 
 				Expect(k8sClient.Delete(context.TODO(), ns)).Should(Succeed())
+
+				Expect(k8sClient.Delete(context.TODO(), dmProfile)).Should(Succeed())
```

Comment:
Is the 'ns' in the previous line the same namespace that this 'dmProfile' is in? Do these two need to be swapped?

Reply:
Yes. I'll swap them because it makes sense. However, envtest doesn't [support deleting a namespace](https://github.com/kubernetes-sigs/controller-runtime/issues/880), so it probably doesn't matter until it gets fixed.